integration/setup/setup.go

```diff
@@ -122,6 +122,7 @@ func installResources(ctx context.Context, config Config) error {
 		// implemented for the unique instance.
 		opts := helmclient.InstallOptions{
 			ReleaseName: key.AppOperatorUniqueName(),
+			Wait:        true,
```

Comment:
The tests were failing due to the status webhook because coredns was not ready yet. 

Waiting for both operators to be ready makes the tests more reliable.

Maybe we should add wait support to app CRs? As it could be useful in some situations. I'll add a note to our board to discuss.

Reply:
How did you fix coredns issue with helmclient's `wait` option? It seems fine for me but just don't get how it fixed this issue.

Reply:
There was a race condition because the test was running before coredns was fully up.

In the chart-operator logs it was failing to call the webhook URL. When I checked the coredns logs its because it wasn't up yet. Waiting for the operator pods to be ready adds some delay but makes the test more reliable.